connector_magento/components/importer.py

```diff
@@ -270,7 +270,11 @@ class DelayedBatchImporter(AbstractComponent):
 
     def _import_record(self, external_id, job_options=None, **kwargs):
         """ Delay the import of the records"""
-        delayable = self.model.with_delay(**job_options or {})
+        if job_options is None:
+            job_options = {}
+        if "channel" not in job_options:
+            job_options["channel"] = "root.magento"
```

Comment:
Well, the only thing that made me not use `queue.job.function` was that, there is no record for `queue.job.channel`, only channel name was used in the decorator. So after update, there would be two channels with the same name.

But now, when I think of it, existing jobs can be updated via migration script and it will be better to have the defaults in `queue.job.function`